corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -67,6 +71,15 @@ hqDefine("cloudcare/js/formplayer/menus/api", [
                         return;
                     }
                 }
+                let appId = params.appId;
+                if (!params.preview) {
+                    if (app.get('copy_of') && app.get('copy_of') === params.appId) {
```

Comment:
nit: I think making a variable `const = app.get('copy_of')` and using that would be nice